pkg/postgres/pgutils/utils.go

```diff
@@ -33,6 +36,19 @@ var (
 	pgxPoolDSNRegex = regexp.MustCompile(`(^| )(pool_max_conns|pool_min_conns|pool_max_conn_lifetime|pool_max_conn_idle_time|pool_health_check_period)=\S+`)
 )
 
+// Logger for Raw SQL portion of migration.
+type printSQLLogger struct {
+	logger.Interface
+}
+
+func (l *printSQLLogger) Trace(ctx context.Context, begin time.Time,
+	fc func() (sql string, rowsAffected int64), err error) {
+
+	sql, _ := fc()
+	fmt.Println(sql + ";")
```

Comment:
Does it have to go to stdout, or should it go through the standard logger (`log.Info` or `log.Log`) ?

Reply:
I'm waffling on whether this should be logged as `Debug` or not.  Seems like a lot of details to log and expose each time central starts.

Reply:
Those are not going to be logged in central at all by default, since one have to explicitly enable DryRun mode, which we don't do. As a follow up to this PR I'm going to post a small tool, which is going to use this functionality. The idea is to print out in exactly the same way as GORM does this, without any logging formatting and what not -- so that one can redirect stdout into a file and then apply it separately as plain sql.